css-layout-classes/style.css

```diff
@@ -0,0 +1,24 @@
+.container {
+  max-width: 800px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-third {
+  flex-basis: calc(100% / 3);
+}
+
+.column-one-fourth {
+  flex-basis: 25%;
+}
+
+.column-second {
```

Comment:
`column-two-thirds` would be a better name

Reply:
I don't understand .... what is two-thirds means here?

Reply:
A third is one portion of something divided into three parts. This class is for an element that is taking up two-thirds of an available area, which makes it a good name to denote the size this class creates.